packages/vue/src/components/sign-up.vue

```diff
@@ -16,9 +16,6 @@
           <user-name-alias />
           <sign-up-password-control />
           <sign-up-confirm-password-control />
-          <base-box data-ui-error v-if="error">
-            {{ error }}
-          </base-box>
```

Comment:
Why was this removed?

Reply:
This was derived from sold old code that was on line 150 for a watcher. With the new redesign this actually was broke, I believe. There is already an error tracked elsewhere on this page, so I don't think we need 2 of them.

Reply:
For now, let's leave the `challengeName` in the TypeScript file. For some reason the plugin I'm using likes it. Let's set a follow up task to troubleshoot it for the future.